### DISCUSSION TITLE
Which maven repository to get Keycloak 15.1.0 artifacts ?

### DISCUSSION THREAD

**Discussion by fidgi** [keycloak/keycloak#9098] [Q&A] [answered]
Hi,

I can't find Keycloak version 15.1.0 artifacts on maven repositories such as https://mvnrepository.com/artifact/org.keycloak .

Is there any maven repo where I can get these artifacts ?

Regards,

**Comment by stianst** [comment, marked as answer]
There's an issue with sync from JBoss Nexus syncing to Maven central. They are looking at resolving that at the moment. In the meantime you can grab these directly from JBoss Nexus.
